src/photos/ducks/clustering/settings.js

```diff
@@ -0,0 +1,35 @@
+import { cozyClient, log } from 'cozy-konnector-libs'
+import { DOCTYPE_PHOTOS_SETTINGS } from '../../../drive/lib/doctypes'
+import {
+  SETTING_TYPE,
+  DEFAULT_SETTING,
+  DEFAULT_PERIOD,
+  DEFAULT_MODE
+} from './consts'
+
+export const createDefaultSetting = async () => {
```

Comment:
There's no `await` here so we can ditch the `async` ;)

```suggestion
export const createDefaultSetting = () => {
```